ote/resources/public/language/fi.edn

```diff
@@ -1141,4 +1141,4 @@
                                     :removeDisabled "Poistotila on poissa käytöstä"}}
                 :handlers {:edit {:tooltip {:text "Siirrä satamia/laitureita klikkaamalla niitä ja liikuttamalla samalla hiirtä"
                                             :subtext "Mikäli haluat peruuttaa muutokset, valitse \"peruuta\"."}}
-                           :remove {:tooltip {:text "Merkitse poistettavaksi klikkaamalla"}}}}}
+                           :remove {:tooltip {:text "Merkitse satama poistettavaksi klikkaamalla."}}}}}
```

Comment:
These marine traffic related leaflet translations seem to be in a generic :leaflet-edit translation tree. This should be named in more specific manner.

Reply:
Moved them under :leaflet-port-edit key